Tools/LyTestTools/tests/unit/test_o3de_editor_test.py

```diff
@@ -9,1138 +9,55 @@
 import pytest
 import unittest.mock as mock
 
-import ly_test_tools
 import ly_test_tools.o3de.editor_test as editor_test
-import ly_test_tools.launchers.exceptions
+from ly_test_tools.launchers import launcher_helper
 
 pytestmark = pytest.mark.SUITE_smoke
 
 
-class TestEditorTest(unittest.TestCase):
+class TestEditorTestBase(unittest.TestCase):
 
-    def test_EditorSharedTest_Init_CorrectAttributes(self):
+    def test_EditorSharedTest_Init_CorrectDefaultAttributes(self):
         mock_editorsharedtest = editor_test.EditorSharedTest()
+        assert mock_editorsharedtest.timeout == 180
+        assert mock_editorsharedtest.test_module is None
+        assert mock_editorsharedtest.attach_debugger is False
+        assert mock_editorsharedtest.wait_for_debugger is False
         assert mock_editorsharedtest.is_batchable
         assert mock_editorsharedtest.is_parallelizable
 
```

Comment:
I don't see much value in asserting about these constants. These tests only verify the class can construct and that nobody touched the constants.

I think it points to good design that there's no extra incidental logic that needs unit testing in this class. Needing to rely on integ tests show that these classes are as data-container-like as possible, and that the previous code was successfully moved into `multi_test_framework.py`